### PR TITLE
Fix flowchart height in Kubernetes Legacy Package blog

### PR DESCRIPTION
**Pull request by saschagrunert**
Fixing the height of the flowchart, which seems to small:

screenshot

### CODE REVIEW

**Comment by saschagrunert** [comment]
@xmudrii PTAL, looks better now on my machine:

screenshot

File: content/en/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ with maintainers to let them know about this change.
 The following diagram shows who's affected by this change in a visual form
 (click on diagram for the larger version):
```
Comment (sftim) [inline content/en/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:61]: Try this instead:
```suggestion
_(click on diagram for the larger version)_
```

**Review by sftim — COMMENTED** [review]
I don't agree with having the diagram shown full size. Instead, let's teach readers about clicking for a big version.

File: content/en/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md (hunk 1 above)
Comment (saschagrunert) [reply content/en/blog/_posts/2023-08-31-legacy-package-repository-deprecation/index.md:61]: Oops, I should have read that, thank you! Will close this PR for now.

**Comment by saschagrunert** [comment]
> I don't agree with having the diagram shown full size. Instead, let's teach readers about clicking for a big version.

Closing the PR since clicking was intentional.

I think it adds more barriers personally, though.

**Pull request closed without merging**